src/main/java/org/black_ixx/bossshop/core/conditions/BSConditionTypeClientVersion.java

```diff
@@ -0,0 +1,41 @@
+package org.black_ixx.bossshop.core.conditions;
+
+import net.minecraft.server.v1_13_R2.EntityPlayer;
+import net.minecraft.server.v1_13_R2.NetworkManager;
+import net.minecraft.server.v1_13_R2.PlayerConnection;
+import org.black_ixx.bossshop.core.BSBuy;
+import org.black_ixx.bossshop.core.BSShopHolder;
+import org.bukkit.craftbukkit.v1_13_R2.entity.CraftPlayer;
+import org.bukkit.entity.Player;
+
+import java.lang.reflect.Field;
+
+public class BSConditionTypeClientVersion extends BSConditionTypeNumber{
+    @Override
+    public void enableType() {}
+
+    @Override
+    public String[] createNames() {
+        return new String[]{"client", "version"};
+    }
+
+    @Override
+    public boolean dependsOnPlayer() {
+        return true;
+    }
+
+    @Override
+    public double getNumber(BSBuy shopitem, BSShopHolder holder, Player p) throws NoSuchFieldException, IllegalAccessException {
+        return getPlayerProtocolVersion(p);
+    }
+
+    private static int getPlayerProtocolVersion(Player player) throws NoSuchFieldException, IllegalAccessException {
+        CraftPlayer craftPlayer = (CraftPlayer) player;
```

Comment:
We really can't be using version dependent NMS classes like this. If we want to use NMS, we're going to need to use reflection.